### PR TITLE
Dependency upgrade 15th Feb

### PR DESCRIPTION
**Pull request by ob6160**
## What does this change?
Upgrades our dependencies to the latest versions

Fixes an issue with upgrading `@babel/preset-react` mentioned in #1453 by including `@emotion/babel-preset-css-prop` as a preset to the Storybook babel config [^1]

[^1]: https://github.com/storybookjs/storybook/issues/7540#issuecomment-934216060

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,24 +33,24 @@
...
     "@guardian/eslint-plugin-source-react-components": "^4.0.2",
     "@guardian/node-riffraff-artifact": "^0.3.2",
-    "@storybook/addon-actions": "^6.5.0-alpha.34",
-    "@storybook/addon-essentials": "^6.5.0-alpha.34",
-    "@storybook/addon-links": "^6.5.0-alpha.34",
-    "@storybook/builder-webpack5": "^6.5.0-alpha.34",
-    "@storybook/manager-webpack5": "^6.5.0-alpha.34",
-    "@storybook/react": "^6.5.0-alpha.34",
+    "@storybook/addon-actions": "^6.4.19",
+    "@storybook/addon-essentials": "^6.4.19",
```
[the remark this answers is not in the document]
Comment (ob6160) [reply package.json:49]: ah yes, good spot - thanks

[the remark this answers is not in the document]
Comment (ob6160) [reply package.json]: the Storybook tests did run successfully though 🤔

[the remark this answers is not in the document]
Comment (ob6160) [reply package.json]: also, aren't you supposed to be on holiday :)

[the remark this answers is not in the document]
Comment (liamness) [reply package.json]: I suppose if the js being borked doesn't cause any visual issues, then the ui tests would still pass.

**Review by liamness — APPROVED** [review]
Tested locally, looks good to me

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ob6160) [reply package.json]: good point 👍

**Pull request merged**